DiagnosticsExtension/Models/ConnectionStringValidator/Constants.cs

```diff
@@ -0,0 +1,64 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Constants.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DiagnosticsExtension.Models.ConnectionStringValidator
+{
+    public static class Constants
+    {
+        public const string User = "User";
+        public const string System = "System";
+        public const string UnderscoreSeperator = "__";
+        public const string ColonSeparator = ":";
+        public const string ClientId = "__clientId";
+        public const string Credential = "__credential";
+        public const string QueueServiceUri = "__queueServiceUri";
+        public const string ValidCredentialValue = "managedidentity";
+        public const string FullyQualifiedNamespace = "__fullyQualifiedNamespace";
+        public const string UnknownErrorSummary = "Validation of connection string failed due to an unknown error.";
```

Comment:
Can we consider moving these wordings to frontend project? It will be easier to maintain all the wordings together instead of crossing multiple projects.

Reply:
This was done intentionally as there is a lot of context that would have to be passed back in the response to the frontend and the logic would then be split between frontend and backend which makes it harder to maintain and more importantly harder to deploy due to the coupling between the two.  Supporting managed identity scenarios is what has added this new complexity.